xdcs-frontend/src/app/view/task-creation/task-creation.component.ts

```diff
@@ -87,14 +112,60 @@ export class TaskCreationComponent {
   }
 
   submitTask() {
+    this.convertFiles()
+      .then(() => this.startTask());
+  }
+
+  convertFiles() {
+    return Promise.all(
+      Array.from(this.files.keys()).map(paramName => this.convertFile(paramName))
+    );
+  }
+
+  convertFile(paramName: string) {
+    return BlobUtils.toString(this.files[paramName], 'utf-8')
```

Comment:
To UTF-8 nie wygląda dobrze, powinieneś zamieniać bloba bez jakiegokolwiek kodowania

Reply:
przerobione na base64